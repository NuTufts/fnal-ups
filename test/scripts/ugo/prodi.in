Product:          ProdI
Flavor:           IRIX+5.3
                  IRIX+5
                  IRIX
                  NULL
Qualifiers:       Butcaption,youhavespaces
DB:               /home/t2/fagan/ups_database
Chains:           current
