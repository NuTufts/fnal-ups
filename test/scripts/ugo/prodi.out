Product:          ProdI
Flavor:           IRIX+5.3
                  IRIX+5
                  IRIX
                  NULL
Qualifiers:       Butcaption,youhavespaces
DB:               /thunder
                  	
                  /lightening
Chains:           current
